kooplex-proxy/proxy/main.py

```diff
@@ -27,42 +27,42 @@ def read_root():
 
 
 @app.get("/country_samples/", response_model=List[schemas.MView_CountrySamples])
-def read_country_samples(skip: int = 0, limit: int = 100):
-    country_samples = crud.get_country_samples(next(get_db()), skip=skip, limit=limit)
+def read_country_samples(schema_key='', skip: int = 0, limit: int = 100):
```

Comment:
Are you planning to change schema_key='' to something similar to what you did in crud.py?

Reply:
is this case our schema_key will be required parameter for the endpoints without using the default schema name

Reply:
Oh I see. Please ignore my comments.